### PR TITLE
[IMP] web: some autocomplete improvements

### PR DESCRIPTION
**Pull request by ged-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/core/autocomplete/autocomplete.js [hunk 1]
```diff
@@ -4,8 +4,7 @@ import { useForwardRefToParent, useService } from "@web/core/utils/hooks";
 import { useDebounced } from "@web/core/utils/timing";
 import { getActiveHotkey } from "@web/core/hotkeys/hotkey_service";
 import { usePosition } from "@web/core/position_hook";
-
```
Comment (Polymorphe57) [inline addons/web/static/src/core/autocomplete/autocomplete.js:7]: commit message:
two small improvement**s**

File: addons/web/static/tests/core/autocomplete_tests.js [hunk 2]
```diff
@@ -338,4 +338,20 @@ QUnit.module("Components", (hooks) => {
...
+            <AutoComplete value="'Hello'"
+                sources="[{ options: [{ label: 'World' }, { label: 'Hello' }] }]"
+                autofocus="true"
+                onSelect="() => {}"
+            />
+        `;
+
+        await mount(Parent, target, { env });
+
+        assert.strictEqual(target.querySelector("input"), document.activeElement);
```
Comment (Polymorphe57) [inline addons/web/static/tests/core/autocomplete_tests.js:355]: I would prefer a more precise selector

**Comment by ged-odoo** [comment]
@mcm-odoo pr updated

**Comment by Polymorphe57** [comment]
robodoo r+

**Pull request closed without merging**